### ISSUE TITLE
Table rendered with empty <td>

### ISSUE THREAD

**Issue by nivv** [thephpleague/commonmark-ext-table#8]
Hey!

Noticed that if there is a closing `|` the extension will output an extra empty `<td>`

Example:

```markdown
| Th 1 | Th 2 | Th 3 |     | 
|------|------|------|-----| 
| td1  | td2  | td3  | td4 | 
```

Outputs: 
```html
<table>
<thead>
<tr>
<th>Th 1</th>
<th>Th 2</th>
<th>Th 3</th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>td1</td>
<td>td2</td>
<td>td3</td>
<td>td4</td>
<td></td>
</tr>
</tbody>
</table>
```

If I change the markdown and remove the ending pipe it works as expected:

```markdown
| Th 1 | Th 2 | Th 3 |     | 
|------|------|------|-----|
| td1  | td2  | td3  | td4 <!-- removed pipe here -->
```

The reason I'm asking this is because many other markdown renderers accept the above behaviour, maybe it's not part of the spec that this extension is referring to.

**Comment by hason** [comment]
The problem is with whitespaces at the end of line.

**Comment by hason** [comment]
Fixed in 0.7 version.
